libconfluo/confluo/container/sketch/frequency_functions.h

```diff
@@ -0,0 +1,35 @@
+#ifndef CONFLUO_CONTAINER_SKETCH_FREQUENCY_FUNCTIONS_H
+#define CONFLUO_CONTAINER_SKETCH_FREQUENCY_FUNCTIONS_H
+
+namespace confluo {
+namespace sketch {
+
+/**
+ * These are monotonically increasing frequency-domain functions.
+ * in f and O(f^2), where f is the frequency of the data point
+ * @tparam counter_t type of input
+ */
+template<typename arg_t = int64_t>
```

Comment:
Are these the only `f` functions we support for universal sketches? How will users provide their own (in the future)?

Reply:
I'll add that functionality in a separate PR, wanted to keep things simple in this one